debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/PostgresToYbValueConverter.java

```diff
@@ -28,17 +28,22 @@ public void configure(Properties props) {
     public void converterFor(RelationalColumn column,
             ConverterRegistration<SchemaBuilder> registration) {
 
+        LOGGER.info("Processing converter for column: {}, type: {}, JDBC type: {}", column.name(), column.typeName(), column.jdbcType());
```

Comment:
do we need these logs as info ones? I think we can keep these as debug.

Reply:
yeah, fair. I think these are useful for development, when we need to figure out the exact type of the column that we're dealing with..